src/civetweb.c

```diff
@@ -192,7 +180,25 @@ clock_gettime(int clk_id, struct timespec *t)
 	}
 	return -1; /* EINVAL - Clock ID is unknown */
 }
+
+/* if clock_gettime is declared, then __CLOCK_AVAILABILITY will be defined */
```

Comment:
Unfortunately I have no OSX development system available. I can test OSX code in Travis CI, but I cannot change the OSX version they are using.
Is it "only" the comment, that's not correct, or is the patch itself not working?